travis-scripts/script.sh

```diff
@@ -20,7 +20,7 @@ INSTDIR=$HOME/cf_install
 
 # Unshallow the clone. Fetch the tags from upstream even if we are on a
 # foreign clone. Needed for determine-version.sh to work.
-git fetch --unshallow
+git fetch --unshallow --no-recurse-submodules
```

Comment:
FYI, I was seeing the same errors yesterday and opened https://tracker.mender.io/browse/ENT-8819
I worked around the error in my PR by increasing the depth for cloning.

This link will go bad when I squash: https://github.com/cfengine/core/pull/4966/commits/6568481e124b4c586d4dca13432d3c4c63897888

```
diff --git a/.travis.yml b/.travis.yml
index 4dcfc5ef5e..40bd9ece7d 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,5 @@
+git:
+  depth: 100
 language: c
 os: linux
 dist: xenial
 ```